frontend/src/text_resources.js

```diff
@@ -197,6 +197,8 @@ let LOCALIZE = new LocalizedStrings({
 
       //Test tabs
       tabs: {
+        menuName: "eMIB Tab Menu",
+        instructionsMenu: "Instuctions Menu",
```

Comment:
Should we put these two strings under **ariaLabel**, since they are meant to be _aria-label_ even if they are set through a propTypes?

In that way, it'll be clear that all ariaLabel are in the same place if we need to change strings related to the narrator.

Same comment for French version of these strings..

Reply:
Also, there is a typo in **"Instructions Menu"**. You forgot the **'r'**.

Reply:
resolved in 94f471a and c0f3343